### PR TITLE
Change GridOut::write_svg() defaults

### PR DESCRIPTION
**Pull request by bangerth**
The GridOut::write_svg() function now no longer outputs the
cell level and number by default; this also obviates the need for the
colorbar and the caption. This is achieved by changing the defaults in
the constructor of the GridOutFlags::Svg structure. This likely comes
closer to what most users want to see from this function -- namely,
just the mesh for inclusion into text.

It might be easiest to read the four commits individually.

### CODE REVIEW

**Comment by bangerth** [comment]
Ping?

File: include/deal.II/grid/grid_out.h [hunk 1]
```diff
@@ -669,41 +669,65 @@ namespace GridOutFlags
...
+     * The style used for the background of the mesh.
+     */
     Background background;
 
     // View angles for the perspective view of the grid; Default is 0, 0 (top
     // view).
     /**
      * The azimuth angle measured from ??? in degrees. Default is 0.
      */
+
```
Comment (tjhei) [inline include/deal.II/grid/grid_out.h]: remove?

**Comment by bangerth** [comment]
I've addressed @tjhei's one comment.

**Pull request merged**